backend/devices/bitbox02/device.go

```diff
@@ -218,15 +224,18 @@ func (device *Device) DeviceInfo() (string, error) {
 
 	response, err := device.query(request)
 	if err != nil {
-		return "", err
+		return nil, err
 	}
 
-	deviceInfo, ok := response.Response.(*messages.Response_DeviceInfo)
+	deviceInfoResponse, ok := response.Response.(*messages.Response_DeviceInfo)
 	if !ok {
-		return "", errp.New("Failed to retrieve device name")
+		return nil, errp.New("Failed to retrieve device info")
 	}
+	deviceInfo := &DeviceInfo{}
```

Comment:
not important, just  fyi: you can init directly like `&DeviceInfo{ Name: ..., Version: ... }`.